cartography/intel/aws/ecs.py

```diff
@@ -0,0 +1,590 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+import boto3
+import neo4j
+
+from cartography.util import aws_handle_regions
+from cartography.util import camel_to_snake
+from cartography.util import dict_date_to_epoch
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_ecs_clusters(boto3_session: boto3.session.Session, region: str) -> Tuple[List, List]:
```

Comment:
Nit: can you type out the extra characters for `Tuple[List[Dict[str, Any]], List[Dict[str, Any]]]`? Same comment for the other typehints.